### PR TITLE
fix avatar size in chat and place them in correct position

### PR DESCRIPTION
**Pull request by NingyuanXu**
- [ ] All features are tested
- [ ] Version is bumped
- [ ] Changelog is updated
- [ ] Documentation is updated if needed

### CODE REVIEW

File: public/stylesheets/chat_room_style.css [hunk 1]
```diff
@@ -55,6 +55,18 @@
...
+  width: 80px;
+  float: left;
+}
+
+img.conversation-avatar-sent {
+  height: 80px;
+  width: 80px;
+  float: right
+}
+
```
Comment (ZijiaZhang) [inline public/stylesheets/chat_room_style.css:69]: We can just use `conversation-avatar` and reuse the `sent` etc.

Comment (NingyuanXu) [reply public/stylesheets/chat_room_style.css:69]: but then how do we differentiate receive and sent

Comment (ZijiaZhang) [reply public/stylesheets/chat_room_style.css:69]: Here https://github.com/ZijiaZhang/CPSC436_Project/blob/11d79325f104bd56d6f08319005fcef1fde8b88c/public/stylesheets/chat_room_style.css#L44

Comment (NingyuanXu) [reply public/stylesheets/chat_room_style.css:69]: better keep as what I wrote before, change .sent will not work

**Pull request merged**